content/zh_tw.html

```diff
@@ -386,6 +387,98 @@
 
 <article>
 
+<h1><a name="expect">語法 expect 與 should</a></h1>
+
+<!-- <p>
+On new projects always use the <code>expect</code> syntax.
+</p> -->
+
+<p>
+在 RSpec3 總是使用 <code>expect</code> 語法。
```

Comment:
I use this translation because in gem transpec mention that
> With Transpec you can upgrade your RSpec 2 specs to RSpec 3 in no time.

so I use the word RSpec3 instead of new projects.